src/app/parser.rs

```diff
@@ -1788,6 +1788,38 @@ impl<'a, 'b> Parser<'a, 'b>
         }
         Ok(())
     }
+    
+    pub fn add_env(&mut self, matcher: &mut ArgMatcher<'a>) -> ClapResult<()> {
+        macro_rules! add_val {
+            ($_self:ident, $a:ident, $m:ident) => {
+                if let Some(ref val) = $a.v.env {
```

Comment:
I ended up not trying to dedup this with default. I found it was going to be annoying. If it's required for commit, I can, but otherwise I'd prefer to leave as is.